### PR TITLE
Title feature added

### PR DESCRIPTION
**Pull request by Agostinelli**
A new feature that allows you to quick read the title of the crawled website (if alive) has been added.
When a given link is passed with the 'u' flag, the program will return the h1 title of the page along with the url. 
 
A little refactor throughout the project has been carried out, in particular the "yield" construct at line 72 and 74 has been removed and replaced it with a simple return. 
To be honest I couldn't understand exactly why the programmer used this construct in the first place, therefore I want to point out here in order to be sure that I haven't broken anything. 

The feature has been tested thoroughly.
Regression testing has been made and it is okay.

### CODE REVIEW

**Comment by PSNAppz** [comment]
@Agostinelli Why did u change the name of function readPage() to read_first_page() ?

**Comment by Agostinelli** [comment]
@PSNAppz because I needed a function without all the print statements. Therefore I copy-pasted readPage() and edited to read_first_page(), and kept the old one as read_page(). 

I know it is not a good solution, but in order to refactor it I needed to ask you first.

**Comment by PSNAppz** [comment]
@Agostinelli Yes, please do refactor the code and commit the changes.

**Comment by Agostinelli** [comment]
What do you mean by refactor exactly?

File: modules/pagereader.py [hunk 1]
```diff
@@ -50,6 +50,46 @@ def readPage(site, extension=False):
         raise("There has been a connection error after three attempts.")
 
 
+def read_page(site, extension=False):
```
Comment (PSNAppz) [inline modules/pagereader.py]: This should be `read_first_page` right?

Comment (Agostinelli) [reply modules/pagereader.py]: No. It's correct. The only problem is that there is a bit of repeated code.

Comment (PSNAppz) [reply modules/pagereader.py]: Ok

**Pull request merged**